Traceback (most recent call last):
  File "d:\Projects\temp\vorticity_MDNN_dg.py", line 72, in <module>
    pinn_nvs = train_vorticity_dg(config,device=device)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Projects\temp\nv_files\train_nvs.py", line 467, in train_vorticity_dg
    dg_NVs = Vorticity(config=config, device=device)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Projects\temp\nv_files\NavierStokes.py", line 111, in __init__
    self.init_M()
    ^^^^^^^^^^^
  File "D:\Users\chris\anaconda3\Lib\site-packages\torch\nn\modules\module.py", line 1688, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'Vorticity' object has no attribute 'init_M'
